### PR TITLE
Update notes

### PR DESCRIPTION
**Pull request by MemoonaTahira**
I have created a PR with just the notes added. I will update the environment.md as you said and create a separate PR.

### CODE REVIEW

**Comment by alexeygrigorev** [comment]
Can you please resolve the conflict? Also if you could do a rebase before PR that would be nice too

**Comment by MemoonaTahira** [comment]
I think it should be good now.

**Comment by MemoonaTahira** [comment]
Can I ask one question, I see that I am again one commit behind. Do I need to keep updating my branch with new commits until it is accepted?

I had synced the fork before PR, but each time another PR is accepted, my branch gets out of date with potential conflicts.

Sorry because this is my first time with a pull request.

File: course-zoomcamp/02-regression/README.md [hunk 1]
```diff
@@ -32,5 +32,4 @@ Did you take notes? You can share them here (or in each unit separately)
 * [Notes from froukje](https://github.com/froukje/ml-zoomcamp/blob/main/week2/Lecture_2_car_price_prediction.ipynb)
 * [Notes from Jon Areas](https://github.com/jxareas/Machine-Learning-Bookcamp-2022/blob/master/notes/02-regression.md)
 * [Notes from Memoona Tahira](https://github.com/MemoonaTahira/MLZoomcamp2022/blob/main/Notes/Notes_for_Chapter_1-Linear_Algebra.ipynb)
-* [Notes from Wesley Barreto](https://github.com/wgb-10/ML-Zoomcamp-2022/blob/main/Session-Projects/02-Regression/my-notebook.ipynb)
```
Comment (alexeygrigorev) [inline course-zoomcamp/02-regression/README.md:35]: Can you please undo it? You're removing someone's else notes

**Comment by alexeygrigorev** [comment]
There are too many commits for a single line change. Please rebase and create a new PR

**Comment by MemoonaTahira** [comment]
> There are too many commits for a single line change. Please rebase and create a new PR

Sure. Wesley added one of my updates to his PR, and you've merged it. Created a new PR for the other remaining update to linear algebra notes.

**Pull request closed without merging**